src/styles/theme.ts

```diff
@@ -250,6 +250,12 @@ const initTheme = (darkMode: boolean) => {
         defaultProps: {
           fullWidth: true,
         },
+        styleOverrides: {
+          root: {
+            // Keystone modals have a z-index of 999
+            zIndex: 998,
+          },
+        },
```

Comment:
This now hides the mobile pairing QR code behind the onboard modal again.

Reply:
Good catch, I've lowered the `z-index` of onboard and it works as expected now:

![image](https://user-images.githubusercontent.com/20442784/202708630-46ce2c56-853e-41c8-bc22-01390525c28a.png)